Makefile

```diff
@@ -32,7 +32,7 @@ ASFLAGS           =
 LDFLAGS           =    -Wl,-pie -specs=$(DEVKITPRO)/libnx/switch.specs -g
 LINKS             =    -limgui -lstbi -ldeko3d -lnx
 
-PREFIX            =    aarch64-none-elf-
+PREFIX            =    $(DEVKITPRO)/devkitA64/bin/aarch64-none-elf-
```

Comment:
```suggestion
PREFIX            =    aarch64-none-elf-
```

Reply:
This should be added to your PATH by the makefile later.

Reply:
Unfortunately I cannot compile the project without this change (make: aarch64-none-elf-g++: No such file or directory). I can not explain exactly the cause of the problem, but it is resolved by specifying the path. By the way, I'm using macOS 10.15.5.

Reply:
That's weird, probably a macOS quirk.